Packs/CommunityCommonScripts/Scripts/BulkCloseIncidents/README.md

```diff
@@ -0,0 +1,38 @@
+Prerequisite:
```

Comment:
This README.md file appears to be manually created. Please use the `demisto-sdk generate-docs` command to automatically generate the documentation for this script. The generated documentation should include command examples.